modules/ROOT/pages/db/database-documentation.adoc

```diff
@@ -1001,9 +1025,9 @@ Selects from a table at a regular interval and generates one message per each ob
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Initial Buffer Size a| Number | The amount of instances that is initially be allowed to be kept in memory to consume the stream and provide random access to it. If the stream contains more data than can fit into this buffer, then the buffer expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. Default value is 100 instances. |  |
-| Buffer Size Increment a| Number | This is by how much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. Default value is 100 instances. |  |
-| Max Buffer Size a| Number | The maximum amount of memory to use. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower than or equal to zero means no limit. |  |
+| Initial Buffer Size a| Number | The number of instances that are initially allowed to be kept in memory to consume the stream and provide random access to it. If the stream contains more data than can fit into this buffer, then the buffer expands according to the *Buffer size increment* attribute, with an upper limit of *Max in memory size*. The default value is 100 instances. | `100`  | 
+| Buffer Size Increment a| Number | Specifies by how much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, in which case a `STREAM_MAXIMUM_SIZE_EXCEEDED` error is raised when the buffer gets full. The default value is 100 instances. | `100`  | 
```

Comment:
I am not sure whether we should use asterisk or backticks for these since this is not a UI.